identifiers/country-fr/oversea.csv

```diff
@@ -0,0 +1,12 @@
+id,name
```

Comment:
If with want to use different terms for the oversea territories with deparment and collectivity we have to define 
Nouvelle-Calédonie as a "sui generis collectivity".
Or we define all under the term oversea_territory? (as suggestion) 

Source:
https://en.wikipedia.org/wiki/Overseas_territory_(France)

Reply:
Good point. I would like to have a common type for all overseas divisions, But I am not sure if we can use Overseas territory for all of them, according to the wikipedia page : 
overseas territory ... is currently only applied to the French Southern and Antarctic Lands.

I am also curious if there

Reply:
..mhhh, you're right. But if we dont find a common type for all, then i would continue to work with the exact types like you started (darpartmant, collectivities, and so on)?

Reply:
I think so, @jloutsenhizer what do you think ?

Reply:
Yeah since it sounds like there's not a common term that can be used for all of them, the more specific ones should be fine.

Reply:
Another solution should be : using region for all the overseas divisions, since according to wikipedia:

- Regarding oversea department : https://en.wikipedia.org/wiki/Regions_of_France : France is divided into 18 administrative regions , of which 13 are located in metropolitan France (i.e. on the European continent), while the other five are overseas regions
- Regarding overseas collectivity : https://en.wikipedia.org/wiki/Overseas_collectivity : The French overseas collectivities, like the French regions, are first-order administrative divisions of France, but have a semi-autonomous status. 

WDYT ?

Reply:
@jloutsenhizer @sguenther85  can you have a look :) ?

Reply:
So wold the proposal to use the region type for the 18 administrative regions and then keep oversea_collectivity? If so  I think that makes sense to me.

Reply:
I was thinking to use region also for oversea collectivity since they are in the same administrative order as region. So, I would have 23 regions.

Reply:
sorry for the long break (vacation). That with the 23 regions would be possible, but then we have also to create a second level below these regions:

 (Wikipedia: Most administrative regions also have the status of regional "territorial collectivities", which comes with a local government, with departmental and communal collectivities below the region level.)

So it would not solve the problem, but only adjust the structure of the oversea regions to the others ;) or not?

So i think we should maybe all agree with the more specific ones "right now" and think about a second time next french parliamentary elections. 
So that we can use these current ocd for the senates elections.
What do you think @azuser @jloutsenhizer ?

Reply:
Looking over it again, I think the current OCD IDs for the overseas regions/collectivities LGTM and we can use as is.